<launch>
	<node name="planner_service" pkg="multiagent_planning" type="planner_node" required="true"/>

	<node name="rviz_visualizer" pkg="rviz" type="rviz"/>

	<group ns="client1">
		<node name="planner_client" pkg="multiagent_planning" type="planner_client" args="1 5 6 0">
			<remap from="/planned_path" to="/client1/planned_path"/>
		</node>
	</group>

	<group ns="client2">
		<node name="planner_client" pkg="multiagent_planning" type="planner_client" args="2 2 8 180">
			<remap from="/planned_path" to="/client2/planned_path"/>
		</node>
	</group>
</launch>